src/buttons/blue-button.jsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import { css, withStyles } from '../default/with-styles';
+
+
+class BlueButton extends React.Component {
```

Comment:
Use ``stateless`` component if no additional ``store`` or ``functional`` implementation is required.